packages/itwinui-css/src/style/component.scss

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) Bentley Systems, Incorporated. All rights reserved.
```

Comment:
Why this file was added when `variables.scss` is left?

Reply:
I don't know; I'm guessing an auto-merge mistake
Fixed